src/main/resources/db/migration/mysql/V1.13__agency.sql

```diff
@@ -37,5 +37,3 @@ ALTER TABLE `organisational_unit`
 ADD CONSTRAINT `FK_organisational_unit_agency_token` FOREIGN KEY (`agency_token_id`) REFERENCES `agency_token` (`id`) ON DELETE CASCADE ON UPDATE CASCADE;
 
 SET FOREIGN_KEY_CHECKS=1;
-
```

Comment:
Do you have to write another sql file to remove them?  (as opposed to deleting part of an already existing file).
i.e. Will Flyway get upset, as this has been deployed to integration already?

Reply:
These scripts haven't made it to production yet so it is more reasonable just to change the original. It will probably cause a conflict on integration (depending on what has been released there) but can be resolved by clearing the db and rebuilding.

Reply:
Yeah I know, I agree.  I was just saying it will cause issues on integration, which will have to be resolved manually.  I would prefer that than going to production with both the add and the remove.

Reply:
Agreed, I'm going to keep the migration as is and resolve any flyway issues manually on integration.